src/lib.rs

```diff
@@ -597,8 +597,7 @@ impl Socket {
 
     /// Send a `&[u8]` message.
     pub fn send(&self, data: &[u8], flags: i32) -> Result<()> {
```

Comment:
I've now added that, including more `From` implementation for `Message`, and simplifying call sites accordingly.